openam-clientsdk/dependency-reduced-pom.xml

```diff
@@ -67,23 +67,23 @@
         <configuration>

```

Comment:
This file should never have been committed in the first place. This file is generated during the build, hence it should be removed and added to .gitignore